package-lock.json

```diff
@@ -506,7 +506,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
+      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```

Comment:
dont know why you had this file in this PR? @brettz9  i reverted

Reply:
I just wanted to be transparent with the result I was given with my npm installs... I don't know if this somehow automatically occurred for me being as I am in China and/or on VPN...

Reply:
yea if this is a problem in the future let me know, i just prefer `https` for all things

Reply:
Sure, me too... I guess this issue was occurring for others too... May have been because of some issue with recursive updating, as I did so to make sure we had the latest packages...